projects/storefrontlib/cms-components/navigation/navigation/navigation-ui.component.ts

```diff
@@ -117,12 +129,6 @@ export class NavigationUIComponent implements OnInit, OnDestroy {
       typeof navNode.url === 'string' &&
       this.winRef.nativeWindow?.location.href.includes(navNode.url)
     ) {
-      this.elemRef.nativeElement
```

Comment:
Any changes to existing methods are considered breaking changes so we should feature flag any functionality addition/modification/removal. Even if its a small change :/

Reply:
Added feature flags + task
https://jira.tools.sap/browse/CXSPA-5919